docs/control_center_docs/control_center_start_guide.rst

```diff
@@ -0,0 +1,740 @@
+4. Here, choose a name and location for your generated defaults file. In this example I will save the file as “example_defaults.json”
+5. You will then be given the option to import the file directly into the Control Center. In order to use this file to configure other modules, you must import it into the Control Center
+
+.. image:: ../_static/control_center_pics/import_defaults_popup.png
+
+6. After clicking Yes, you will see your imported defaults as an option in the dropdown. This file will now always be available in the Control Center
+7. Once in the Control Center, you can use the Set button exactly as you would with a Vertiq supplied Defaults file
+
+Importing a User Defaults File
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+It is also possible to import externally generated Defaults files into the Control Center. To do so:
+
+#. Select the Import option
+#. Use the File Explorer to find the Defaults file you would like to import
+#. You will see the file appear in the dropdown of other Defaults files, and it is ready for use
+
+******************************************
+Testing Modules with the Control Center
+******************************************
+The Control Center provides a basic suite of testing options to ensure that your module is functioning properly. These options are available in the testing tab, 
+available on the left side.
+
+.. note::
+    Before using the Control Center's testing functionality, we recommend that you increase your module's timeout parameter to 1.5s as the Control Center
+    sends heartbeat checks only once per second. This parameter is available through the tuning tab. If you do not increase the timeout value, your module may fail to spin as expected. When 
+    you are finished testing your module, please remember to reset your module's timeout value to your desired value. 
+
+.. warning::
+    Please remove all propellers before interacting with any testing parameters. Failure to do so is dangerous.
+
+.. warning::
+    If you are using a power supply to power your module, it is possible to damage or destroy your module with aggressive commands, e.g. quickly switching from spinning at full speed to stopping. 
+    This is because Vertiq modules can also act as generators. In general, power supplies, unlike batteries, cannot absorb the energy generated by the module. As such, aggressive commands can lead to dangerous 
+    voltage spikes when connected to a power supply. To prevent damage to the module when commanding it aggressively on a power supply, it is recommended to turn on the regeneration voltage protection feature.
+
+In order to set a testing parameter, simply enter a value into the spin box, and hit the down arrow set button. Parameters such as Coast do not have a spin box, 
+and only require that you press the set button to trigger the behavior.
+
+The testing tab for a Vertiq 23-06 2200Kv looks as follows:
+
+.. image:: ../_static/control_center_pics/23_testing_tab.png
+    :scale: 50%
+
```

Comment:
A little bit of intro here to say that you are going to describe some common testing parameters across firmwares is useful to motivate and explain these sections below, since they aren't a 1-to-1 with your picture anymore. That's good that they aren't, since ESC Input is speed-specific, I think its just helpful to give an introduction explaining why you are talking about this specific subset of the testing parameters